src/SFA.DAS.RoatpOversight.Domain/AppealsOutcomeOversightReviews.cs

```diff
@@ -0,0 +1,26 @@
+﻿using SFA.DAS.ApplyService.Types;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SFA.DAS.RoatpOversight.Domain
+{
+    public class AppealsOutcomeOversightReviews
```

Comment:
this feels redundant maybe, I think  where AppealsOutcomeOversightReviews is used, you could replace it with List<AppealsOutcomeOversightReview>

Reply:
Nah it fits with the existing pattern but has incorrect name. Should be something like **AppealOutcomes**

Reply:
agree, do what Martin said

Reply:
ok will rename